src/editor/locale/lang.ru.js

```diff
@@ -216,12 +216,12 @@ export default {
     snapping_stepsize: 'Шаг привязки:',
     grid_color: 'Цвет сетки',
     done: 'Done',
-    change_xxx_color: 'Change xxx color',
+    change_xxx_color: 'Поменять xxx цвет',
```

Comment:
Does not sound as Russian?

Reply:
It's Russian. What does `xxx` mean? I thought it's like a template, isn't it?

Reply:
My mistake. Thank you for the contribution